advocacy_docs/partner_docs/SIBVisionsGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Partner Information'
+description: 'Details for SIB Visions and Vision X'
```

Comment:
Review the entire guide Jasmer and make sure we are using 
SIB VIsions VisionX correctly in all spots.